### ISSUE TITLE
Fine-Tuning Model

### ISSUE THREAD

**Issue by nurul274** [fadilahnurkh09/intprog2-project#21]
Melakukan penyempurnaan model dengan membuka sebagian layer MobileNetV2 agar model dapat menyesuaikan fitur dengan dataset bentuk wajah.

- [ ] Menentukan layer yang akan di-unfreeze
- [ ]  Menentukan learning rate fine-tuning
- [ ]  Melakukan training lanjutan
- [ ]  Membandingkan hasil sebelum dan sesudah fine-tuning
- [ ]  Menyimpan model terbaik
